File: exercicios/1985/1985.py
def entrada():
    vl = 0
    x = int(input())

    for i in range(x):
        v =  input().split()
        cod = int(v[0])
        qt = int(v[1])

        if cod == 1001:
            vl = vl + 1.5 * qt
        elif cod == 1002:
            vl = vl + 2.5 * qt
        elif cod == 1003:
            vl = vl + 3.5 * qt
        elif cod == 1004:
            vl = vl + 4.5 * qt
        elif cod == 1005:
            vl = vl + 5.5 * qt

    return(vl)

def main():
    vl = entrada()
    print('{:.2F}'.format(vl))
main()

File: exercicios/JavaScript/1985/1985.js
var input = require('fs').readFileSync('stdin','utf-8');
var lines = input.split('\n');

var p = parseInt(lines.shift())
var total = 0

for(let i=0;i<p;i++){
    var valores = lines.shift().split(' ')
    var pedido = valores[0]
    var qt = parseInt(valores[1])

    if(pedido == 1001){
        total += qt * 1.50;
    }else if(pedido == 1002){
        total += qt * 2.50
    }else if(pedido == 1003){
        total += qt * 3.50
    }else if(pedido == 1004){
        total += qt * 4.50
    }else if(pedido == 1005){
        total += qt * 5.50
    }
}
console.log(parseFloat(total).toFixed(2))